evm_loader/program/src/entrypoint.rs

```diff
@@ -595,6 +595,36 @@ fn process_instruction<'a>(
             }
             Ok(())
         },
+        EvmInstruction::DeleteAccount { seed } => {
+            let deleted_acc_info = next_account_info(account_info_iter)?;
+            let creator_acc_info = next_account_info(account_info_iter)?;
+            let evm_loader = next_account_info(account_info_iter)?;
```

Comment:
You do not need to pass `evm_loader` id through account meta. You should use `program_id` argument from https://github.com/neonlabsorg/neon-evm/blob/d8acb310b2d3e68ea49af4458dabf915f43c17e8/evm_loader/program/src/entrypoint.rs#L113

Reply:
fixed